scripts/Draconic-Evolution.zs

```diff
@@ -770,16 +770,10 @@ mods.avaritia.ExtremeCrafting.addShaped(<DraconicEvolution:dislocatorReceptacle>
 recipes.addShapeless(<DraconicEvolution:safetyMatch:1000>, [<gregtech:gt.metaitem.01:32471>, <ore:dustDraconium>]);
 
 // --- Distortion Flame
-mods.avaritia.ExtremeCrafting.addShaped(<DraconicEvolution:xRayBlock> * 4, [
-[null, null, null, null, null, null, null, null, null],
-[null, null, null, null, null, null, null, null, null],
-[null, null, null, <ore:paneGlassColorless>, <ore:paneGlassColorless>, <ore:paneGlassColorless>, null, null, null],
-[null, null, <ore:paneGlassColorless>, <ore:gemNetherStar>, <ore:plateDraconium>, <ore:gemNetherStar>, <ore:paneGlassColorless>, null, null],
-[null, null, <ore:paneGlassColorless>, <ore:plateDraconium>, <ore:gemDiamond>, <ore:plateDraconium>, <ore:paneGlassColorless>, null, null],
-[null, null, <ore:paneGlassColorless>, <ore:gemNetherStar>, <ore:plateDraconium>, <ore:gemNetherStar>, <ore:paneGlassColorless>, null, null],
-[null, null, null, <ore:paneGlassColorless>, <ore:paneGlassColorless>, <ore:paneGlassColorless>, null, null, null],
-[null, null, null, null, null, null, null, null, null],
-[null, null, null, null, null, null, null, null, null]]);
+recipes.addShaped(<DraconicEvolution:xRayBlock> * 8, [
+[<minecraft:glass_pane>, <gregtech:gt.metaitem.01:17330>, <minecraft:glass_pane>],
+[<gregtech:gt.metaitem.01:17330>, <gregtech:gt.metaitem.02:29500>, <gregtech:gt.metaitem.01:17330>],
+[<minecraft:glass_pane>, <gregtech:gt.metaitem.01:17330>, <minecraft:glass_pane>]]);
```

Comment:
I'm not sure you understand the point of this. The idea is so that you can see into caves so that you can light them up and reduce lag. If it used nether stars, it would be way too late for that kind of thing, and too hard. And who would use nether stars that are really hard to get for something like this? It's a consumable, you know? Just using one isn't enough, you're liable to use several. Also, I'd say nether star is HV, because I'd want a nano suit, but I'm not the best fighter. Also, if you find some aluminium in chests (which isn't that hard), you can make thaumium in LV. @boubou19 Do you have any comments?

Reply:
Well, as Prometheus said, this should be used to spot the caves around your base. And unless you are good at judging the distances (or if you play with friends), you'll likely need multiples to see if you go in the good direction, if you did not go past the cave you wanted to light. So you'll need more than one for sure. This can also be used to spot mineshafts, but a part from the spotting some chests, i don't see any use for this. Also, if you use this to find lootchests, then you'll need more than 8. I think that players are quickier to craft stuff than spending hours to get the stuff from loots. Changing the diamond to a flawless diamond will make sure that early game players will not be able to spam them, as their only way to get it will be to try their luck on a small diamond ore.

Nether star is definitely too late: nether is so much of a pain that going into it and try to get 3 wither heads is a big no for most of the players. The only way we could get ez wither would be to use a one ring to get ez warp, but it would encourage the use of the one ring, which makes me not a big fan of the idea.

Reply:
the quest book encourages players to go in the nether at LV btw.

Reply:
ok fine if it is consumable lets change this recipe